app/controllers/users_controller.rb

```diff
@@ -17,6 +17,13 @@ def create
 
     if @user.save
 
+      begin
+        Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') unless Rails.env.test?
```

Comment:
You will be learning about using a gem called vcr in the next lesson which will record http request/response cycles. You should actually be letting stripe get hit in test mode too.